CONTRIBUTING.md

```diff
@@ -59,8 +59,10 @@ go into `stubs`. Each subdirectory there represents a PyPI distribution, and
 contains the following:
 * `METADATA.toml`, describing the package. See below for details.
 * Stubs (i.e. `*.pyi` files) for packages and modules that are shipped in the
-  source distribution. If the Python 2 version of the stubs must be kept
-  *separate*, they can be put in a `@python2` subdirectory.
+  source distribution.
+* If the stubs are either Python 2-only, or if the Python 2 and Python 3 stubs
+  are separate, the Python 2 stubs are put in a `@python2` subdirectory. In the
+  former case, there are no stubs or package directories on the top level.
```

Comment:
If I understood this correctly, we could just say this instead, making it more obvious where to find the Python 3 stubs:

```suggestion
  are separate, the Python 2 stubs are put in a `@python2` subdirectory.
  Stubs or package directories at the top level are always used with Python 3,
  so there are no stubs or package directories on the top level of Python 2-only stubs.
```